### ISSUE TITLE
d3ploy library not found 

### ISSUE THREAD

**Issue by gwenchee** [arfc/d3ploy#64]
putting solver into a separate file causes this error when trying to run a cyclus simulation: 
`No module found for path libd3ploy.no_inst.so`

**Comment by gwenchee** [comment]
If we put solver in a separate file, this issue occurs. But if we keep it in no_inst as its own function then we need to find a way to pull out that function to test it

**Comment by FlanFlanagan** [comment]
Are you adding the separate file to the install script?

**Comment by jbae11** [comment]
this might come from my ignorance, but the current setup file grabs everything in the directory, and the `solver.py` is in the same directory as the `no_inst.py`. Are there any additional edits I need to make this so that this works?

**Comment by FlanFlanagan** [comment]
Oh, I forgot I did that. Good work past Robert. 

I think the issue is that the module isn't building. Typically with the python modules that error means that it didn't build properly.

**Comment by jbae11** [comment]
the setup.py output message seems like everything went well, and there are `solver.py` and `no_inst.py` files in `.../anaconda3/lib/python-3.6/site-packages/d3ploy`. I'm not sure what I'm missing.. any ideas?

**Comment by FlanFlanagan** [comment]
Could try and import solver.py in a python terminal ?

**Comment by jbae11** [comment]
I tried that and it worked, and I made sure that the module `solver` doesn't clash with any pre-installed module. I guess one way to solve it would be to integrate the `solver.py` into the `no_inst.py` file, which was the way it was before. This raises the problem of unit tests, though. But ideally I think we should have all the ''solvers'' - deploy_solver, arma, arch, and other solvers out to a separate file maybe? Whats your thought?
